docs/basicuse.rst

```diff
@@ -338,6 +338,45 @@ ability to define usable tokens, but any actor can then reference these tokens.
     2015-01-14 15:02:22,165 INFO      [DRY: Notify Engineering] Sending message "Hey room .. I'm done with the release. Get back to work" to Hipchat room "Engineering"
     2015-01-14 15:02:22,239 INFO      [DRY: Notify Cust Service] Sending message "Hey room .. I'm done with the release. Have a nice day" to Hipchat room "Cust Service"
 
+Contextual tokens stored in separate file
+'''''''''''''''''''''''''''''''''''''''''
+
+When multiple kingping json files need to leverage the same context for different purposes it is useful to put the contexts into a stand alone file and then reference that file. Since `%TOKENS%` won't be parsed inside of the context file they need to be explicitly passed into it.
```

Comment:
Re-word: "Since `%TOKENS%` won't be parsed inside of the context file they need to be explicitly passed into it."
Suggest: "Context files support [tokens](#<link to env token section above>) just like our [Macros](#<link to macro actor>) do -- you must pass in the `%TOKEN%` and its `value`. See below for examples."